### ISSUE TITLE
Write accessibility page in team playbook

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-design-system#3944]
## What

Fill the (currently empty) [accessibility page in the team playbook](https://govuk-design-system-team-docs.netlify.app/accessibility/).

## Why

We should document how we are and should be doing things around accessibility.
That will make it easier for new joiners to learn about it and established team members to remind themselves.

Somewhat related is the ['Testing accessibility concerns' section in the accessibility strategy](https://design-system.service.gov.uk/accessibility/accessibility-strategy/#testing-accessibility-concerns).

## Who needs to work on this

Anika

## Who needs to review this

Any team member

## Done when

- [x] Dependent on https://github.com/alphagov/govuk-design-system/issues/3943
- [x] Raise PR for additions to the page
- [ ] Review PR
- [ ] Publish

**Comment by selfthinker** [comment]
Because this was left untouched for so long, we have decided that I should start with just adding a bunch of links to relevant resources.
I have done that now in this PR: https://github.com/alphagov/design-system-team-docs/pull/175
